### PR TITLE
Th reoder comment scriptsjs

### LINKED ISSUE #18: Review Page Start Up
As a group go through `scripts.js` file:
* start at page load and track what is happening with class instantiation/methods
* find areas that can be structured better

### PR DESCRIPTION
**Pull request by relyt4me**
### What’s this PR do?
Adding comments to all of the methods and walking through the scripts.js file

### Where should the reviewer start?
changes all made to scipts.js

### How should this be manually tested?
changes made that would affect fucntionality are just code order. Can be tested with a npm start and going to page

### Any background context you want to provide?
Walked through this together

### What are the relevant tickets?
resolves #18

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -46,21 +46,23 @@ var userStairsThisWeek = document.getElementById('userStairsThisWeek');
 var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
 var bestUserSteps = document.getElementById('bestUserSteps');
 var streakList = document.getElementById('streakList');
-var streakListMinutes = document.getElementById('streakListMinutes')
+var streakListMinutes = document.getElementById('streakListMinutes');
+
+window.onload = startApp(); //Starts Here (global variables currentInformation = {})
```
Comment (jordy1611) [inline src/scripts.js:51]: good call moving this to the top

File: src/scripts.js [hunk 2]
```diff
@@ -46,21 +46,23 @@ var userStairsThisWeek = document.getElementById('userStairsThisWeek');
...
-  let activityRepo = new Activity(activityData);
-  var userNowId = pickUser();
-  let userNow = getUserById(userNowId, userRepo);
-  let today = makeToday(userRepo, userNowId, hydrationData);
-  let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
-  historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
-  addInfoToSidebar(userNow, userRepo);
+  let hydrationRepo = new Hydration(hydrationData); // created our hydration class based on (need to get hydration from API)
+  let sleepRepo = new Sleep(sleepData); // same as above Sleep
+  let activityRepo = new Activity(activityData); // same as above Activity
```
Comment (jordy1611) [inline src/scripts.js:59]: Are we making these global?

File: src/scripts.js [hunk 3]
```diff
@@ -46,21 +46,23 @@ var userStairsThisWeek = document.getElementById('userStairsThisWeek');
...
+  var userNowId = pickUser(); // rename currentUser = newRandomUser() (userNowID = currentUser.id)
+  let userNow = getUserById(userNowId, userRepo); //covered from above
+  let today = makeToday(userRepo, userNowId, hydrationData); // rename mostRecentDate and assign to '2020/01/22' for now and possibly use a method later to get the most recent date
+  let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData); // a random date? (do we need this) REMOVE
+  historicalWeek.forEach((instance) => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`)); // remove the random week and historical week placements REMOVE
+  addInfoToSidebar(userNow, userRepo); // fills out user infor (iteration 1 dashboard)
   addHydrationInfo(userNowId, hydrationRepo, today, userRepo, randomHistory);
   addSleepInfo(userNowId, sleepRepo, today, userRepo, randomHistory);
   let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
```
Comment (jordy1611) [inline src/scripts.js:68]: Should be in a DOM specific helper function

File: src/scripts.js (hunk 1 above)
Comment (carlymclift) [reply src/scripts.js:51]: Yeah this makes a lot more sense to have it up top.

File: src/scripts.js (hunk 2 above)
Comment (carlymclift) [reply src/scripts.js:59]: I think we are going to try and keep them as it (with some adjustments). Making them global is definitely an option if you feel like that would make more sense to you.

**Comment by carlymclift** [comment]
Awesome notes. This is going to make things a lot easier going forward because now we know how everything is laid out.

**Pull request merged**